.github/workflows/skipped-test-tracker.yml

```diff
@@ -0,0 +1,51 @@
+name: Skipped Test Tracker
```

Comment:
can you add an explanation / comment to this job? Is it tracking which tests are skipped? If so, can you elaborate what the bigger purpose is?